control-plane/controllers/configentries/configentry_controller_test.go

```diff
@@ -1750,7 +1750,7 @@ func TestConfigEntryControllers_doesNotCreateUnownedConfigEntry(t *testing.T) {
 
 			// Change the config entry so protocol is https instead of http if test case says to
 			if c.makeDifferentFromConsul {
-				svcDefaults.Spec.Protocol = "https"
+				svcDefaults.Spec.Protocol = "http2"
```

Comment:
**NOTE**: This changes is not relevant to this PR, currently until tests fail with error 
`Unexpected response code: 500 (1 error occurred:* invalid value for protocol: https)` , this change fixes it. Need to investigate further as to why `https` is an invalid protocol for ServiceDefault config